src/styles/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
```

Comment:
Could you remove flex-box for the card block?
There are several blocks (they will be arranged vertically by default) inside it and there is no need to use flex here. It may cause some problem in your tests.